pkg/ocicni/ocicni.go

```diff
@@ -239,20 +245,15 @@ func initCNI(exec cniinvoke.Exec, cacheDir, defaultNetName string, confDir strin
 		cacheDir:     cacheDir,
 	}
 
-	if exec == nil {
-		exec = &cniinvoke.DefaultExec{
```

Comment:
Assigning `exec` here does not have any effect. I assume we would like to set the `exec` as part of the `plugin`.

Reply:
About `exec` argument:

> NewCNIConfigWithCacheDir returns a new CNIConfig object that will search for plugins in the given paths use the given exec interface to run those plugins, or if the exec interface is not given, will use a default exec handler

Reply:
Not sure what the default handler would be and whether it matters, though.

Reply:
I assume it should be the moved code. But assigning it to `exec` makes no sense at all.